content/08.transmission.md

```diff
@@ -0,0 +1,160 @@
+
+Several studies on disparities in COVID-19 have compared groups in terms of their probability of a severe outcome (like hospitalization or death) given that a person is diagnosed.
+Early in the pandemic, it is more difficult to study or identify the larger population of all exposed or all infected people because of the existence of undetected infections; over the course of the pandemic, indirect estimates of infection rates through modeling improve and more direct estimates through a combination of serological tests, sampling methods and statistical methods have been made possible through biotechnology developments.
+In contrast, [@doi:10.1038/s41586-020-2521-4] were able to compare overall rates of death from COVID-19, not simply in the subsample of diagnosed individuals, thus combining the probability of infection with the probability of death given infection. <!-- TODO: In contrast with what? -->
+This was done using a dataset of 17 million adults in the United Kingdom.
+In this administrative dataset, [@doi:10.1038/s41586-020-2521-4] found that male sex, older age, economic deprivation, nonwhite ethnicity (principally Black or South Asian), obesity, diabetes,  asthma, and several other comorbid conditions were associated with a higher risk of death from COVID-19.
+They cautioned that despite the large sample size, their observational data did not directly provide information on the causal mechanisms which led to these statistical associations.
+However, the ethnic and economic disparities did not vanish when adjusting for observed preexisting conditions.
+[@doi:10.1038/s41586-020-2521-4] also found that the predictive value of socioeconomic status seemed to increase over time, perhaps because many higher-paid workers were able to transition to working online.
+
+Besides the direct harms caused by infection, many populations are in indirect risk of serious harm due to the social and economic effects of the pandemic and of the efforts needed to fight it.
+These might include individuals with substance use disorder [@doi:10.7326/M20-1141], victims of natural disasters [@doi:10.1038/s41558-020-0804-2], and victims of human trafficking [@doi:10.1016/j.eclinm.2020.100409].
+The pandemic could also delay the fight against other major infectious diseases, such as HIV, malaria and tuberculosis, potentially leading to a further increase of mortality [@doi:10.25561/78670].
+Although they are beyond the scope of this paper, further research is needed in order to prevent these harms. <!-- there are already some policy recommendations in literature that we cite - we should probably direct folks there. I think we probably want a stronger synthesis and closing than this -->
+
+### Clinical Presentation of COVID-19
```

Comment:
I feel like this should be in pathogenesis.

Reply:
#b710e95 should handle this!